Apromore-Custom-Plugins/Process-Simulation-Info-Logic/src/main/java/org/apromore/processsimulation/service/SimulationInfoService.java

```diff
@@ -120,12 +120,12 @@ private void deriveGeneralSimulationInfo(
             ((double) (endTimeMillis - startTimeMillis) / (double) 1000) / (double) logSummary.getCaseCount());
 
         builder.processInstances(logSummary.getCaseCount())
-            .currency(Currency.valueOf(defaultCurrency.toUpperCase()))
+            .currency(Currency.valueOf(defaultCurrency.toUpperCase(Locale.ENGLISH)))
             .startDateTime(Instant.ofEpochMilli(logSummary.getStartTime()).toString())
             .arrivalRateDistribution(
                 Distribution.builder()
-                    .timeUnit(TimeUnit.valueOf(defaultTimeUnit.toUpperCase()))
-                    .type(DistributionType.valueOf(defaultDistributionType.toUpperCase()))
+                    .timeUnit(TimeUnit.valueOf(defaultTimeUnit.toUpperCase(Locale.ENGLISH)))
+                    .type(DistributionType.valueOf(defaultDistributionType.toUpperCase(Locale.ENGLISH)))
```

Comment:
Perhaps it'd be worth creating a DOCUMENT_LOCALE constant with the value Locale.ENGLISH?

Reply:
Noted. Will make this change in my next PR